Kitodo/src/main/resources/messages/messages_en.properties

```diff
@@ -11,7 +11,14 @@
 AND=Contains
 ERROR=ERROR
 AuswaehlenAusVorhandenenProzessen=Choose from previous processes
-authorizations=Authorizations
+Automatische\ Generierung\ der\ SICI=Automatic generation of SICI
```

Comment:
This key is never used and white spaces should be avoided in translation keys - so remove it.